frontend/src/components/utils/NavBar/index.tsx

```diff
@@ -96,45 +136,83 @@ const NavBar = ({ headersData }: Props): ReactElement => {
-          {...{
-            component: RouterLink,
-            to: href,
-            color: 'inherit',
-            style: { textDecoration: 'none' },
-            key: label,
-          }}
-        >
-          <MenuItem>{label}</MenuItem>
-        </Link>
+        <ThemeProvider theme={muiTheme}>
+          <Link
+            {...{
+              component: RouterLink,
+              to: href,
+              color:
+                (location.pathname === '/' && label.includes('Home')) ||
+                ((location.pathname.includes('landlord') ||
+                  location.pathname.includes('reviews')) &&
+                  label.includes('Reviews'))
+                  ? 'secondary'
+                  : 'primary',
+              style: { textDecoration: 'none' },
+              key: label,
+            }}
+          >
+            <MenuItem>{label}</MenuItem>
+          </Link>
+        </ThemeProvider>
       );
     });
   };
 
   const getMenuButtons = (): ReactElement[] => {
     return headersData.map(({ label, href }) => {
       return (
-        <Button
-          {...{
-            key: label,
-            color: 'inherit',
-            to: href,
-            component: RouterLink,
-            className: menuButton,
-          }}
-        >
-          {label.toUpperCase()}
-        </Button>
+        <ThemeProvider theme={muiTheme}>
+          <Button
+            {...{
+              key: label,
+              color:
+                (location.pathname === '/' && label.includes('Home')) ||
+                ((location.pathname.includes('landlord') ||
+                  location.pathname.includes('reviews')) &&
+                  label.includes('Reviews'))
+                  ? 'secondary'
+                  : 'primary',
+              to: href,
+              component: RouterLink,
+              className: menuButton,
+            }}
+          >
+            {label.toUpperCase()}
+          </Button>
+        </ThemeProvider>
       );
     });
   };
 
   const homeLogo: ReactElement = (
-    <Typography variant="h4" component="h1" className={logo}>
-      <Link color="textPrimary" underline="none" href="/">
-        CUAPTS
-      </Link>
-    </Typography>
+    <Grid container xs={11} md={7} direction="column">
+      <Grid item>
+        <Grid container alignItems="center">
+          <Grid item>
+            <Link color="textPrimary" underline="none" href="/">
+              <Icon className={icon}>
+                <img src={LogoIcon} alt="CU Apts Logo" height={57.41} width={30.16} />
+              </Icon>
+            </Link>
+          </Grid>
+          <Grid item>
+            <Typography variant="h4" component="h1" className={logo}>
+              <Link color="textPrimary" underline="none" href="/">
+                CUAPTS
+              </Link>
+            </Typography>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid item>
+        <Typography variant="h4" component="h1" className={description}>
```

Comment:
In terms of UI, you probably want to make the text a little smaller because right now the slogan almost seems like the same size as the logo which also makes the logo look too small.  

Also, we should horizontally align the logo with the "Search Renting Companies" stuff 
<img width="1107" alt="Screen Shot 2021-05-10 at 3 23 47 PM" src="https://user-images.githubusercontent.com/57203639/117714094-72943300-b1a4-11eb-8712-545228f6b8a2.png">